src/dataServices/apiDataService.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable class-methods-use-this */
+
+import {
+    ConditionalDeleteResourceRequest,
+    CreateResourceRequest,
+    DeleteResourceRequest,
+    GenericResponse,
+    PatchResourceRequest,
+    Persistence,
+    ReadResourceRequest,
+    ResourceNotFoundError,
+    UpdateResourceRequest,
+    vReadResourceRequest,
+} from 'fhir-works-on-aws-interface';
+import axios from 'axios';
+
+export class ApiDataService implements Persistence {
+    updateCreateSupported: boolean = false;
+
+    // TODO: Pull this value from AWS Param Store
+    static readonly INTEGRATION_TRANSFORM_URL = 'http://localhost:4000';
+
+    async createResource(request: CreateResourceRequest): Promise<GenericResponse> {
+        let genericResponse: GenericResponse = { message: '', resource: {} };
+        try {
+            const url = `${ApiDataService.INTEGRATION_TRANSFORM_URL}/${request.resourceType}`;
+            const response = await axios.post(url, request.resource);
+            genericResponse = { message: '', resource: response.data.resource };
+        } catch (e) {
+            console.error(e);
+            this.handleError(e);
+        }
+        return Promise.resolve(genericResponse);
```

Comment:
just `return genericResponse `

async methods wrap the return value in a promise. There's no point in "double-promise-wrapping" the return value

Reply:
sounds good